### PR TITLE
Update ETC.tsv

### PR DESCRIPTION
**Pull request by mamoruk**
Recipe

### CODE REVIEW

**Comment by Sunaries** [comment]
What is the reason for the closure?

**Comment by mamoruk** [comment]
:stuck_out_tongue_closed_eyes:
sorry it's my fault

File: ETC.tsv [hunk 1]
```diff
@@ -18919,127 +18919,127 @@ ETC_20151224_018920
...
+ETC_20151224_018943	Recipe - Richie Bow
+ETC_20151224_018944	Recipe - Superior Richie Bow
+ETC_20151224_018945	Recipe - Windlass
+ETC_20151224_018946	Recipe - Cradu Shooter
+ETC_20151224_018947	Recipe - Superior Cradu Shooter
+ETC_20151224_018948	Recipe - Sledgehammer
+ETC_20151224_018949	Recipe - Burawa
+ETC_20151224_018950	Recipe - Superior Burawa
+ETC_20151224_018951	Recipe - Chauve-souris 
+ETC_20151224_018952	Recipe - Corseska
```
Comment (Sunaries) [inline ETC.tsv]: Corsesca (I've changed it in my pending PR in ITEM)

File: ETC.tsv [hunk 2]
```diff
@@ -18919,127 +18919,127 @@ ETC_20151224_018920
...
+ETC_20151224_018953	Recipe - Superior Corseska
```
Comment (Sunaries) [inline ETC.tsv]: Corsesca (I've changed it in my pending PR in ITEM)

Comment (mamoruk) [inline ETC.tsv]: and line ITEM_20151224_008591   $Superior Staff----->$Superior Cross Staff

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by Sunaries** [comment]
You have merging conflicts. I've changed " $Superior Staff----->$Superior Cross Staff"

**Comment by mamoruk** [comment]
merging conflicts? how to fix?

**Comment by Sunaries** [comment]
Your branch needs to be updated with the merged PRs. If you can't do this, I'll add it in for you since I will create a new PR with fixes in a day or two.

**Pull request closed without merging**